### PR TITLE
Multiple updates to multiple Dutch translation files.

### PR DESCRIPTION
**Pull request by ly-nld**
Dutch translation for YouTube.

### CODE REVIEW

**Comment by repfuel** [comment]
A lot of grammatical fixes, better wording and overal I think this improvement will be beneficial for Dutch SudoMemo users. Please let me know if it is sufficient or if I am missing anything.

**Comment by repfuel** [comment]
I think for now there are the most major ones. I am sure there is still a lot more that has to be corrected / translated. But I'll most likely do that in another PR when I got the time for that.

**Comment by ly-nld** [comment]
Leaving another comment for this PR.

File: nl_NL/LC_MESSAGES/discordbot.po [hunk 1]
```diff
@@ -112,8 +112,8 @@ msgstr ""
 
 msgid "USAGE_link"
 msgstr ""
-"   link\n"
-"      Link je Sudomemo account met Discord"
+"   koppel\n"
```
Comment (meemo) [inline nl_NL/LC_MESSAGES/discordbot.po:115]: Line 115 should be unchanged, the command key doesn't change with localization.

**Review by meemo — CHANGES_REQUESTED** [review]
Sorry for taking a while, and thanks for the translations! Just need 1 change.

File: nl_NL/LC_MESSAGES/discordbot.po (hunk 1 above)
Comment (ly-nld) [reply nl_NL/LC_MESSAGES/discordbot.po]: Changed it back to "link". My bad.

**Review by meemo — APPROVED** [review]

**Comment by meemo** [comment]
Thanks!

**Pull request merged**